cppcache/test/TXIdTest.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <TXId.hpp>
+
+#include <gtest/gtest.h>
+
+namespace apache {
+namespace geode {
+namespace client {
+
+TEST(TXIdTest, testIncrementTransactionID) {
+  auto tx = new TXId();
+  ASSERT_EQ(1, tx->getId());
+
```

Comment:
need `delete(tx);` ?

Reply:
Use `std::unique_ptr` if this needs to even be a pointer. Probably just as testable as a stack allocated variable.

Reply:
Thanks. I have changed it to stack allocated variables.